src/routes/file_info_index/file_info_index_blueprint.py

```diff
@@ -29,7 +29,8 @@ def create_update_dataset_file_infos(dataset_id):
             # Specify argument to confirm dataset_id is for a Dataset entity.
             theDataset = fworker.get_entity(entity_id=dataset_id, entity_type_check='DATASET')
 
-            fworker.verify_op_permission(aDataset=theDataset)
+            if not fworker.verify_user_in_write_group(aDataset=theDataset) and not fworker.verify_data_admin():
```

Comment:
Is Data-Admin group access required for this PUT `/datasets/<dataset_id>/reindex` call? I don't know the exact use case.

Reply:
Data-Admin group members **_can_** reindex a dataset, but more commonly it will be done by a regular user who is in the write group of the specified Dataset. So it is **_not_** a requirement.

Reply:
I understand. I had thought this PUT `/datasets/<dataset_id>/reindex` call can be triggered by any token as long as it belongs to the `HuBMAP-Read` group. Because essentially what it does is to fetch the entity details and reindex in Elasticsearch by updating the document to the latest version. 

Another question is will this reindex call by triggered using an internal token (the secret key modified)? @shirey can you clarify?